Source/MQTTnet.Extensions.WebSocket4Net/WebSocket4NetMqttChannel.cs

```diff
@@ -52,6 +52,12 @@ public async Task ConnectAsync(CancellationToken cancellationToken)
                 }
             }
 
+#if NET452 || NET46 || NET461 || NET462
+        var sslProtocols = _webSocketOptions?.TlsOptions?.SslProtocol ?? SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
```

Comment:
In my opinion we should only allow 1.2 by default. 1.1 is clearly obsolete and should be activated by users actively, knowing the risk.

Reply:
I second that. IETF declared 1.0 and 1.1 "historic" last year, it's time to move on.

Also, you can enable 1.3 even in net 4.5/4.6 using a numeric enum constant. A bit hacky, but better IMHO.

Reply:
From the issue and here combined: 
@chkr1011 What do you think of 1.3 as numeric value?

I will add NET47 to the filter.